pyproject.toml

```diff
@@ -2,43 +2,40 @@
 name = "expipe_plugin_cinpla"
 version = "0.1.5"
 authors = [
-  { name="Mikkel Lepperod", email="mikkel@simula.no" },
-  { name="Alessio Buccino", email="alessiop.buccino@gmail.com" },
+    { name = "Mikkel Lepperod", email = "mikkel@simula.no" },
+    { name = "Alessio Buccino", email = "alessiop.buccino@gmail.com" },
 ]
 
 description = "Expipe plugins for the CINPLA lab."
 readme = "README.md"
-requires-python = ">=3.9"
+requires-python = ">=3.10"
 classifiers = [
     "Programming Language :: Python :: 3",
     "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
 ]
 
 dependencies = [
-    "expipe>=0.6.0", 
+    "expipe>=0.6.0",
     "neuroconv>=0.4.6",
     "pyopenephys>=1.2.0",
     "spikeinterface[full,widgets]>=0.100.0",
     "pynwb>=2.5.0",
-    "neuroconv>=0.4.6",
     "ipywidgets>=8.1.1",
     "nwbwidgets>=0.11.3",
-    "tbb>=2021.11.0",
+    "tbb>=2021.11.0",                        # TODO: pip can't find tbb or tbb4py (at least on macOS). Is it needed?
     "pynapple>=0.5.1",
-    "spython>=0.3.13",
+    "spython>=0.3.13",                       # TODO: is this needed?
```

Comment:
Since `spython` is for having a Python API for working with Singularity containers, I do not see really see why it has to be a hard requirement. Is it even used? If so, can it be an optional dependency?

Reply:
same for `spython`: the idea would be to use it on EduCloud

Reply:
but currently it's not used, so this could be dropped!

Reply:
Removed in [8d8e408](https://github.com/CINPLA/expipe-plugin-cinpla/pull/72/commits/8d8e408ddc4b67c9d7ee3f919de3706320bf09f7). We can just add it again if we start using it :)